Receiver/Receiver.test.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+import Receiver
+
+
+class ReceiverTest(unittest.TestCase):
+  #Test for Sum of last n values
+  def test_for_sum_of_last_value(self):#Positive senario ::sum of last value::n =1
+    test_input=[1,2,3]
+    self.assertTrue(Receiver.sum_of_last_n_values(test_input,1)==3)#Returns->3
+    
+  def test_for_sum_of_last_two_values(self):#Positive senario :: sum of last two value::n = 2
+    test_input=[1,2,3]
+    self.assertTrue(Receiver.sum_of_last_n_values(test_input,2)==5)#Returns->2+3
+
+  def test_for_sum_of_zero_values(self):#Negative senario :: last n values,where n=0
+    test_input=[1,2,3]
+    self.assertTrue(Receiver.sum_of_last_n_values(test_input,0)==0)#Returns->0[no last elements]
+
+  def test_for_input_value_grater_than_length_list(self):#Negative senario :: last n values,where n grater than length of input_list
+    test_input=[1,2,3]
+    self.assertTrue(Receiver.sum_of_last_n_values(test_input,5)==0)#Returns->0[since 'n' is greater then length of list]
+
+  def test_for_sum_of_values_in_empty_List(self):#Negative senario :: Empty list
+    test_input=[]
+    self.assertTrue(Receiver.sum_of_last_n_values(test_input,3)==0)#Returns->0[Empty list]
+ 
+  def test_for_sum_of_negative_values(self):#Positive senario 
+    test_input=[-1,-2,-3]
+    self.assertTrue(Receiver.sum_of_last_n_values(test_input,3)==-6)#Returns->-1-2-3[Empty list]
+    
+ 
+  #Test for Simple moving average of the last n values
+  def test_for_simple_moving_average_of_last_value(self):#Positive senario ::verage last value
+    test_input=[1,2,3]
+    self.assertTrue(Receiver.simple_moving_average_of_n_values(test_input,1)==3) #Returns->3/1
+    
+  def test_for_simple_moving_average_of_last_five_value(self):#Positive senario ::average_of_last_two_value
+    test_input=[1,2,3,4,5]
+    self.assertTrue(Receiver.simple_moving_average_of_n_values(test_input,5)==3)#Returns->(1+2+3+4+5)/5::2.5
```

Comment:
Test is correct, but what's the `::2.5` in the comment?
Also, floating point cannot be tested for equality. In case of numbers like 3.33333.... you will need to provide tolerance. That tolerance is also a documentation of your behavior. Good to capture in tests.